ci/build.py

```diff
@@ -265,6 +277,22 @@ def load_docker_cache(tag, docker_registry) -> None:
     else:
         logging.info('Distributed docker cache disabled')
 
+def push_docker_cache(registry, tag, image_id) -> None:
```

Comment:
We have a docker_cache.py file specifically for that purpose. I'd recommend to make the appropriate changes in there instead of putting it into this file.

Reply:
Yes, I am utilizing docker_cache.py already.

Reply:
Excellent. I'd recommend you to move this logic into that file then.

Reply:
Moved.